packages/toasts/src/en.json

```diff
@@ -0,0 +1,3 @@
+{
+    "DEFAULT_MESSAGE": "Butter"
```

Comment:
I don't think this file is needed?

Reply:
I got an error with the international provider requesting a default message. I believe it should only be on the test environment, but it returned a test error when no default message for language was given.